Notesnook.API/Controllers/MonographsController.cs

```diff
@@ -92,7 +97,7 @@ private async Task<Monograph> FindMonographAsync(string itemId)
         }
 
         [HttpPost]
-        public async Task<IActionResult> PublishAsync([FromBody] Monograph monograph)
+        public async Task<IActionResult> PublishAsync([FromQuery] string deviceId, [FromBody] Monograph monograph)
```

Comment:
Will this continue to work for old users?

Reply:
Oh, I didn't consider that case at all. I think we can just skip the sync process altogether if the `deviceId` param is not present